jmx-metrics/docs/target-systems/hadoop.md

```diff
@@ -0,0 +1,59 @@
+# Hadoop Metrics
+
+The JMX Metric Gatherer provides built in Hadoop metric gathering capabilities.
+These metrics are sourced from: https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-common/Metrics.html
+
+### HDFS Metrics
+* Name: `hdfs.disk.usage`
```

Comment:
all metrics need `hadoop.` prefix added

Reply:
Good catch. This should be addressed now.